docs/New-Maintainer-Checklist.md

```diff
@@ -25,12 +28,12 @@ A few requests:
   button in "create a merge commit" mode for Homebrew/brew or "squash and
   merge" for a single formulae change. If in doubt, check with e.g. GitX that
   you've not accidentally added merge commits
-- still create your branches on your fork rather than in the main repository
-- if still in doubt please ask for help and we'll help you out - these are
-  probably worth a read:
-    - https://github.com/Homebrew/brew/blob/master/docs/Brew-Test-Bot-For-Core-Contributors.md
-    - https://github.com/Homebrew/brew/blob/master/docs/Maintainer-Guidelines.md
-    - possibly everything else in the documentation
+- still create your branches on your fork rather than in the main repository (note GitHub's UI will create edits and reverts on the main repository by default)
```

Comment:
[GitHub](https://help.github.com/desktop/guides/contributing/creating-a-branch-for-your-work/) is using "on your fork" (emphasis mine):

> To propose a change to a repository you've forked, you can create a branch **on your fork** and then send a pull request to a project maintainer.